app/elements/socobo-recipe/socobo-recipe-overview.html

```diff
@@ -508,7 +508,7 @@
           for (var e in el) {
             var listItem = el[e];
             var internalEl = Polymer.dom(listItem.root).querySelector(".check paper-checkbox");
```

Comment:
The `<socobo-element-list-item>` exposes a property `isChecked`. There is no need to reach into it and query the checkbox directly.